src/components/flowchart/components/sliced-pipeline-action-bar/sliced-pipeline-action-bar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from '../../../ui/button';
+
+import './sliced-pipeline-action-bar.scss';
+
+export const SlicedPipelineActionBar = ({
+  chartSize,
+  slicedPipeline,
+  visibleSidebar,
+}) => {
+  const { outerWidth } = chartSize;
+
+  const transformX = visibleSidebar ? outerWidth / 2 + 100 : outerWidth / 2;
+  if (slicedPipeline.length > 0) {
+    return (
+      <div
+        className="sliced-pipeline-action-bar"
+        style={{ transform: `translateX(${transformX}px)` }}
+      >
+        <div className="sliced-pipeline-action-bar--info">
+          {`${slicedPipeline.length} selected`}
+        </div>
+        <div className="sliced-pipeline-action-bar--cta">
+          <Button>Slice</Button>
+        </div>
+      </div>
+    );
+  } else {
+    return null;
+  }
```

Comment:
[Nit suggestion] In general, it would be nice to handle the negative cases first and then handle the positive case. Example -

if (!slicedPipeline || slicedPipeline.length <= 0) {
return null
}
....then the positive case, no need of else

Reply:
Also, if we are returning null here in-case slicedPipeline.length <= 0. Can we mount or call SlicedPipelineActionBar in the parent only when slicedPipeline.length > 0 ? Is there a reason to pass the prop and then check it in the child component ? Thank you

Reply:
yeah good point, originally i was thinking the flowchart is already too big so I'd like to keep the logic inside the child component to keep it tidy. But think we can move it to the parent so dont need to pass the props to the child if dont need to